src/AddressVerification/MatchedAddress.php

```diff
@@ -22,19 +22,17 @@ class MatchedAddress extends Address
      */
     public function __construct(array $matchedAddress)
     {
-        list(
-            $this->name,
-            $this->phone,
-            $this->email,
-            $this->companyName,
-            $this->addressLine1,
-            $this->addressLine2,
-            $this->addressLine3,
-            $this->cityLocality,
-            $this->stateProvince,
-            $this->postalCode,
-            $this->countryCode,
-            $this->addressResidentialIndicator
-            ) = array_values($matchedAddress);
+        $this->name = $matchedAddress['name'];
```

Comment:
Can we nullable `??` all of these in case they are missing?

Reply:
Sure thing. Some of these fields are non-nullable though. I think we want to always assume we get the data back for those: addressLine1, CityLocality, StateProvince, postalCode, countryCode, and addressResidentialIndicator

Reply:
Oh I guess I could make those empty strings in the event they've returned as null.

Reply:
Yeah I just want this to be bulletproof - we don't want library to crash. So empty strings sounds good.

Reply:
This isn't a critical thing, but what we're doing here is deprecated as of PHP 8.2 [[source](https://php.watch/versions/8.2/dynamic-properties-deprecated)], so we will want to declare (and probably type) these properties properly rather than rely on this behavior.

Reply:
Makes sense. Changes made 👍